src/FurCoNZ.Web/ViewModels/TicketTypeViewModel.cs

```diff
@@ -0,0 +1,37 @@
+﻿using FurCoNZ.Web.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace FurCoNZ.Web.ViewModels
+{
+    public class TicketTypeViewModel
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public int PriceCents { get; set; }
+        public int TotalAvailable { get; set; }
+
+        public decimal Price => (decimal)PriceCents / 100;
```

Comment:
Careful with this. It's fine since it lacks a setter, but we should not be calculating anything based off this since floats have all sorts of weird rounding issues. I'd prefer if we moved this into the Razor template itself but not an issue for now.